apps/wallet/src/stacks/Explorer/CollectionStack/Collectible.tsx

```diff
@@ -29,9 +39,8 @@ export const CollectibleFeat = () => {
 	};
 
 	useEffect(() => {
-		// TODO: need to resolve and remove can go back if possible
-		if (!curCollectible && navigationRef.canGoBack()) {
-			// navigateBack();
+		if (!curCollectible) {
```

Comment:
use this kind of hook to check for e.g when to navigation usually a bad idea, it would become extremely complex quickly.